### DISCUSSION TITLE
Update fontawesome

### DISCUSSION THREAD

**Discussion by wssndrf** [hugo-fixit/FixIt#554] [General]
Hello,

I would very much like to link my Bluesky profile on my blog, unfortunately the Bluesky icon is not displayed in the social.yaml despite adjustments on my part. I have found out that this is only included from fontAwesome version 6.5.2 and FixIT currently uses version 6.4.2 as far as I know - can I update this somehow?

I would be very grateful for your help.

**Comment by Lruihao** [comment]
Ok，I will update it.

**Reply by wssndrf** [reply]
This was my source: https://main--isobelbeasley.netlify.app/resources/update-fa-old-hugo/
